modoboa/admin/management/commands/subcommands/_repair.py

```diff
@@ -4,6 +4,7 @@
 
 from django.conf import settings
 from django.core.management.base import BaseCommand
+from django.utils.encoding import smart_str
```

Comment:
smart_str returns str in both python 2 and 3 whereas smart_text returns unicode in python 2 and str in python3. Sometimes, we need smart_str when the API keeps using str in both python 2 and 3....